analyzer/src/funTest/assets/projects/synthetic/pub/no-lockfile/.dart_tool/pub/bin/sdk-version

```diff
@@ -0,0 +1 @@
+2.3.2-dev.0.0.flutter-e3edfd36b2
```

Comment:
I guess this file should not be part of the commit?

Reply:
removed